modules/capture-video/src/main/native/linux/buildlib.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p build
```

Comment:
Running `.buildlib.sh`  works fine, but requires manual intervention. Maybe it can be triggered from the native-gradle task?

Reply:
yes, I tried to do that but all my attempts failed.